cli/dts/lib.deno.ns.d.ts

```diff
@@ -562,7 +562,10 @@ declare namespace Deno {
    */
   export function exit(code?: number): never;
 
-  /** @category Runtime Environment */
+  /**
+   * @tags allow-env
```

Comment:
Would it be possible to further scope this? Ie. some APIs require "full" permissions (eg. `Deno.mainModule()`), while others work fine with allow lists

Reply:
Any further scoping will just make the documentation unreadable, IMO.  I retained the details in the JSDoc.  This is simply to give a visual indicator and to be able to categorise things, versus being exhaustive.